test/other/embind_tsgen.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface Test {
+  functionOne(_0: number, _1: number): number;
+  functionTwo(_0: number, _1: number): number;
+  functionFour(_0: boolean): number;
+  constFn(): number;
+  functionThree(_0: ArrayBuffer|Uint8Array|Uint8ClampedArray|Int8Array|string): number;
+  delete(): void;
+}
+
+export interface MainModule {
+  Test: {new(): Test};
+  global_fn(_0: number, _1: number): number;
+}
```

Comment:
The this output file match the result of `fsfmt`?   (I think it would good if it did).

Reply:
This is using clang-format which is recommended internally.

Reply:
Great!    Just to confirm, the output you are generating is in the clang-format style?    Is there just one clang format style for typescript or is this going of the `.clang-format` config file in the emscripten directory?

Reply:
Yeah, the output is trying to match clang-format. There's an internal test that I'm trying to match. It seems to just be the default format that `clang-format` applies (no .clang-format file).